src/core/thread/meshcop_dataset.hpp

```diff
@@ -166,8 +166,23 @@ class Dataset
 
     ThreadError Set(const otOperationalDataset &aDataset);
 
+    /**
+     * This method removes a TLV from the Dataset.
+     *
+     * @param[in] aType The type of a specific TLV.
+     *
+     */
     void Remove(Tlv::Type aType);
 
+    /**
+     * This method excludes the Timestamp TLV when appending Dataset TLV into a message.
+     *
+     * @retval kThreadError_None    Successfully append Dataset TLV without Timestamp TLV.
+     * @retval kThreadError_NoBufs  Insufficient available buffers to append the message with Dataset TLV.
+     *
+     */
+    ThreadError RemoveTimestamp(Message &aMessage);
```

Comment:
I think renaming this to something like `AppendMleDatasetTlv` would make this more clear.